CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to opamp-go library
+
+## Generate Protobuf Go Files
+
+Make sure you have [Protobuf Compiler](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation)
```

Comment:
Since `make gen-proto` depends on `protoc-gen-gofast`, we should probably include some instruction on how to install it, since this isn't covered in this link, and isn't all that intuitive.

`> go install github.com/gogo/protobuf/protoc-gen-gofast@latest`

Reply:
could the build tools be used here https://github.com/open-telemetry/build-tools/tree/main/protobuf?

Reply:
I didn't mean to depend on gogo proto, let me remove that. I think it is not necessary for now.

Reply:
Changed to use https://github.com/open-telemetry/build-tools/tree/main/protobuf